edge-agent-sdk/src/commonTest/kotlin/org/hyperledger/identus/walletsdk/pollux/PolluxImplTest.kt

```diff
@@ -1065,27 +1065,31 @@ class PolluxImplTest {
                     "VerifiableCredential",
                     "StatusList2021Credential"
                 ],
-                "issuer": "did:prism:f3a762b1f9741f82791fbdafd0ea2bbc7b8720fcc4fba66f9fac8f896fac35d9",
-                "id": "http://192.168.68.113:8000/cloud-agent/credential-status/6c67b6c0-1bd9-47a1-85ee-f88edaa5a894",
-                "issuanceDate": 1718845277,
+                "issuer": "did:prism:8c050b5928592f4dca045ec7cff7feb2066806d0f3de553d1b8548e03d20f2dc",
+                "id": "http://192.168.68.113:8000/cloud-agent/credential-status/5c6dca3f-fa9d-4a13-af4c-fd31fc9d268c",
+                "issuanceDate": 1723817424,
                 "credentialSubject": {
+                    "id": "",
                     "type": "StatusList2021",
                     "statusPurpose": "Revocation",
-                    "encodedList": "H4sIAAAAAAAA_-3BIQEAAAACIJP_JzvDAjQAAAAAAAAAAAAAAAAAAADA2wCGK9BQAEAAAA=="
+                    "encodedList": "H4sIAAAAAAAA_-3BIQEAAAACIBf4f64zLEADAAAAAAAAAAAAAAAAAAAAvA34c39nAEAAAA=="
                 },
                 "proof": {
-                    "type": "EcdsaSecp256k1Signature2019",
+                    "type": "DataIntegrityProof",
                     "proofPurpose": "assertionMethod",
-                    "verificationMethod": "data:application/json;base64,eyJAY29udGV4dCI6WyJodHRwczovL3czaWQub3JnL3NlY3VyaXR5L3YxIl0sInR5cGUiOiJFY2RzYVNlY3AyNTZrMVZlcmlmaWNhdGlvbktleTIwMTkiLCJwdWJsaWNLZXlKd2siOnsiY3J2Ijoic2VjcDI1NmsxIiwia2V5X29wcyI6WyJ2ZXJpZnkiXSwia3R5IjoiRUMiLCJ4IjoiQU5xNHdCZS1QdlY4UXEzN214THl0RnVxTFJlbG51X251SVMyMEhoZTZsYmsiLCJ5IjoiQU9nWkZmQnBmWlZuNWVuLTFBVEV2YnNyOHExMVpUWTZodEJtTC1xbXFRd3cifX0=",
-                    "created": "2024-06-20T01:01:18.063982543Z",
-                    "jws": "eyJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdLCJhbGciOiJFUzI1NksifQ..c9_mSbKooKiER7GBMj6bty8367VbIj3pRCXF_5vsbhjhe7cA4SoNtUn-DNzlwzIM5n-VMptmuWf33PEXtklEQA"
+                    "verificationMethod": "data:application/json;base64,eyJAY29udGV4dCI6WyJodHRwczovL3czaWQub3JnL3NlY3VyaXR5L211bHRpa2V5L3YxIl0sInR5cGUiOiJNdWx0aWtleSIsInB1YmxpY0tleU11bHRpYmFzZSI6InVNRll3RUFZSEtvWkl6ajBDQVFZRks0RUVBQW9EUWdBRTJleWI5bkdPYlhscTJGVFNZUThsaS1Fc2g0RWcwR1FramZUNUFOMVgySnpfMk1rQTRSdHJLODN4Z25ISE1wd0FoTGRKdHhUdHFlZ0lOVXY1SXU5SDV3PT0ifQ==",
+                    "created": "2024-08-16T14:10:24.312462845Z",
+                    "proofValue": "zAN1rKog8b3npWyeQuM3rDqufJJzD9dwKaK7hmqw5fNytzFqKNT5QzkP768aHD1Smn1vFNDDfRfhWNEmFyq7DSDrrKXFCqQAcZ",
+                    "cryptoSuite": "eddsa-jcs-2022"
                 }
             }
         """
 
         pollux = PolluxImpl(apollo, castorMock, api)
-
         assertTrue(pollux.checkEncodedListRevoked(revocationRegistryJson, 1))
+        assertTrue(pollux.checkEncodedListRevoked(revocationRegistryJson, 2))
+        assertTrue(pollux.checkEncodedListRevoked(revocationRegistryJson, 3))
+        assertFalse(pollux.checkEncodedListRevoked(revocationRegistryJson, 4))
```

Comment:
@curtis-h here I'm testing the pollux method that uses BitString to verify the credential revocation status.

Reply:
yes, but this is test is more of an integration test, it lacks the direct assertion made possible with unit tests,  and there is no way of demonstrating the values given should yield the assertions being made